src/render-qr-codes.ts

```diff
@@ -1,21 +1,71 @@
-const { createCanvas } = require('canvas');
-const fs = require('fs');
-const readline = require('readline');
-const QrCode = require('./qr-code');
+/**
+ * Nimiq QR Code Generator
+ * Generates SVG QR codes with Nimiq styling and gradient options.
+ * 
+ * Features:
+ * - Creates SVG-based QR codes
+ * - Supports Nimiq's brand colors and gradients
+ * - Configurable error correction levels
+ * - Custom fill options including radial/linear gradients
+ * - CLI interface for single QR code generation
+ * 
+ * The generator supports both programmatic usage and CLI operation.
+ */
 
+import { createCanvas, Canvas, CanvasRenderingContext2D } from 'canvas';
```

Comment:
`Canvas` and `CanvasRenderingContext2D` are used as types only.
```suggestion
import { createCanvas, type Canvas, type CanvasRenderingContext2D } from 'canvas';
```

(also in `render-coins.ts`)